code/EventLinkQR/app/src/main/java/com/example/eventlinkqr/OrgHomePage.java

```diff
@@ -0,0 +1,94 @@
+package com.example.eventlinkqr;
+
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.AdapterView;
+import android.widget.Button;
+import android.widget.ListView;
+
+import androidx.navigation.Navigation;
+
+import com.google.firebase.Timestamp;
+import com.google.firebase.firestore.CollectionReference;
+import com.google.firebase.firestore.QueryDocumentSnapshot;
+
+import java.util.ArrayList;
+import java.util.Date;
+
+public class OrgHomePage extends Fragment {
+
+    ListView eventList;
+    EventArrayAdapter eventsAdapter;
+    ArrayList<Event> dataList;
+    Button createEventButton;
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+            Bundle savedInstanceState) {
+
+        View view = inflater.inflate(R.layout.org_home_page, container, false);
+        eventList = view.findViewById(R.id.event_list_view);
+        createEventButton= view.findViewById(R.id.create_event_button);
+        //String [] events = {"event1", "event2", "event3", "event4", "event5"};
```

Comment:
This should probably be removed, or should have a comment if it is still used for testing stuff locally.